### PR TITLE
Readme - Minecraft Edu Updates

### PR DESCRIPTION
**Pull request by leestott**
Included links to minecraft education and details that all students get Minecraft edu if they have a office365 address

### CODE REVIEW

**Review by salmanmkc — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -56,15 +56,33 @@ Here are some examples of the content and star speakers we have this year:
...
 mee03@minecrafteducationedition.com  
 ...  
 mee99@minecrafteducationedition.com  
 Pick one, doesn't matter
 Password for all: Build2020  
-**NOTE: THESE MAY NOT BE AVAILABLE PAST BUILD**
+
+### Minecraft Education 
+
+-You can find more details of [Minecraft education](https://education.minecraft.net/)
```
Comment (salmanmkc) [inline README.md]: Here I would refer to it as Minecraft: Education Edition as was done above

File: README.md [hunk 2]
```diff
@@ -56,15 +56,33 @@ Here are some examples of the content and star speakers we have this year:
...
+-Install [Minecraft Education](https://education.minecraft.net/get-started/)
```
Comment (salmanmkc) [inline README.md]: Same as last, refering to as Minecraft: Education Edition

File: README.md (hunk 1 above)
Comment (drguthals) [reply README.md]: Agreed, Thanks @salmanmkc !

Comment (drguthals) [inline README.md]:
```suggestion
-You can find more details of [Minecraft: Education Edition](https://education.minecraft.net/)
```

File: README.md [hunk 3]
```diff
@@ -56,15 +56,33 @@ Here are some examples of the content and star speakers we have this year:
...
 ...  
 mee99@minecrafteducationedition.com  
 Pick one, doesn't matter
 Password for all: Build2020  
-**NOTE: THESE MAY NOT BE AVAILABLE PAST BUILD**
+
+### Minecraft Education 
+
+-You can find more details of [Minecraft: Education Edition](https://education.minecraft.net/)
+-Install [Minecraft Education](https://education.minecraft.net/get-started/)
```
Comment (drguthals) [inline README.md]:
```suggestion
-Install [Minecraft: Education Edition](https://education.minecraft.net/get-started/)
```

File: README.md [hunk 4]
```diff
@@ -56,15 +56,33 @@ Here are some examples of the content and star speakers we have this year:
...
 mee99@minecrafteducationedition.com  
 Pick one, doesn't matter
 Password for all: Build2020  
-**NOTE: THESE MAY NOT BE AVAILABLE PAST BUILD**
+
+### Minecraft Education 
+
+-You can find more details of [Minecraft: Education Edition](https://education.minecraft.net/)
+-Install [Minecraft: Education Edition](https://education.minecraft.net/get-started/)
+
```
Comment (drguthals) [inline README.md:88]:
```suggestion
  
### Educational Content on Minecraft
```

File: README.md [hunk 5]
```diff
@@ -56,15 +56,34 @@ Here are some examples of the content and star speakers we have this year:
...
 Password for all: Build2020  
-**NOTE: THESE MAY NOT BE AVAILABLE PAST BUILD**
+
+### Minecraft Education 
+
+-You can find more details of [Minecraft: Education Edition](https://education.minecraft.net/)
+-Install [Minecraft: Education Edition](https://education.minecraft.net/get-started/)
+  
+### Educational Content on Minecraft
+Minecraft has released a series of free educational sessions that parents can download to help children learn at home during COVID-19
```
Comment (drguthals) [inline README.md]:
```suggestion
[Minecraft](https://www.minecraft.net/) has released a series of free educational sessions that parents can download to help children learn at home during COVID-19.
```

File: README.md [hunk 6]
```diff
@@ -56,15 +56,34 @@ Here are some examples of the content and star speakers we have this year:
...
+
+### Minecraft Education 
+
+-You can find more details of [Minecraft: Education Edition](https://education.minecraft.net/)
+-Install [Minecraft: Education Edition](https://education.minecraft.net/get-started/)
+  
+### Educational Content on Minecraft
+[Minecraft](https://www.minecraft.net/) has released a series of free educational sessions that parents can download to help children learn at home during COVID-19.
+
+Minecraft Marketplace, which features items and add-ons that people can use in the popular block-building game, has a new Education section.
```
Comment (drguthals) [inline README.md]:
```suggestion
[Minecraft Marketplace](https://www.minecraft.net/marketplace), which features items and add-ons that people can use in the popular block-building game, has a new Education section.
```

File: README.md [hunk 7]
```diff
@@ -56,15 +56,34 @@ Here are some examples of the content and star speakers we have this year:
...
+-Install [Minecraft: Education Edition](https://education.minecraft.net/get-started/)
+  
+### Educational Content on Minecraft
+[Minecraft](https://www.minecraft.net/) has released a series of free educational sessions that parents can download to help children learn at home during COVID-19.
+
+[Minecraft Marketplace](https://www.minecraft.net/marketplace), which features items and add-ons that people can use in the popular block-building game, has a new Education section.
+
+The educational content lets players explore the International Space Station, through a partnership with NASA, as well as learn to code with a robot, visit Washington DC landmarks, find and build 3D fractals, learn what it’s like to be a marine biologist and more. This is launching for free download today and will be available through June 30, 2020. [Minecraft FREE Educational Session](https://news.microsoft.com/en-gb/2020/03/25/minecraft-launches-free-educational-content-for-children/)
+
+**NOTE: THE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BEABLE TO LOG INTO MINECRAFT EDUCATION**
```
Comment (drguthals) [inline README.md]:
```suggestion

```

File: README.md [hunk 8]
```diff
@@ -56,15 +56,33 @@ Here are some examples of the content and star speakers we have this year:
...
+
+
+### You can try out the Minecraft: Education Edition with these usernames:  
 mee01@minecrafteducationedition.com  
 mee02@minecrafteducationedition.com  
 mee03@minecrafteducationedition.com  
 ...  
 mee99@minecrafteducationedition.com  
 Pick one, doesn't matter
 Password for all: Build2020
```
Comment (drguthals) [inline README.md:75]:
```suggestion
Password for all: Build2020  
  
**NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT EDUCATION**
```

File: README.md [hunk 9]
```diff
@@ -56,15 +56,35 @@ Here are some examples of the content and star speakers we have this year:
...
 mee01@minecrafteducationedition.com  
 mee02@minecrafteducationedition.com  
 mee03@minecrafteducationedition.com  
 ...  
 mee99@minecrafteducationedition.com  
 Pick one, doesn't matter
 Password for all: Build2020  
-**NOTE: THESE MAY NOT BE AVAILABLE PAST BUILD**
+  
+**NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT EDUCATION**
```
Comment (drguthals) [inline README.md]:
```suggestion
**NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT: EDUCATION EDITION**
```

File: README.md [hunk 10]
```diff
@@ -56,15 +56,35 @@ Here are some examples of the content and star speakers we have this year:
...
+  
+### Educational Content on Minecraft
+[Minecraft](https://www.minecraft.net/) has released a series of free educational sessions that parents can download to help children learn at home during COVID-19.
+
+[Minecraft Marketplace](https://www.minecraft.net/marketplace), which features items and add-ons that people can use in the popular block-building game, has a new Education section.
+
+The educational content lets players explore the International Space Station, through a partnership with NASA, as well as learn to code with a robot, visit Washington DC landmarks, find and build 3D fractals, learn what it’s like to be a marine biologist and more. This is launching for free download today and will be available through June 30, 2020. [Minecraft FREE Educational Session](https://news.microsoft.com/en-gb/2020/03/25/minecraft-launches-free-educational-content-for-children/)
+
 
 If these are no longer available, we recommend you check out the Minecraft: Education Edition Hour of Code, which is free and doesn't require a login!
```
Comment (drguthals) [inline README.md]:
```suggestion

```

File: README.md [hunk 11]
```diff
@@ -56,17 +56,36 @@ Here are some examples of the content and star speakers we have this year:
...
 mee01@minecrafteducationedition.com  
 mee02@minecrafteducationedition.com  
 mee03@minecrafteducationedition.com  
 ...  
 mee99@minecrafteducationedition.com  
 Pick one, doesn't matter
 Password for all: Build2020  
-**NOTE: THESE MAY NOT BE AVAILABLE PAST BUILD**
+  
+**NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT: EDUCATION EDITION**
```
Comment (drguthals) [inline README.md]:
```suggestion
**NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT: EDUCATION EDITION**  

If these are no longer available, we recommend you check out the [Minecraft: Education Edition Hour of Code](https://education.minecraft.net/lessons/minecraft-hour-of-code/), which is free and doesn't require a login!  
```

File: README.md [hunk 12]
```diff
@@ -56,17 +56,38 @@ Here are some examples of the content and star speakers we have this year:
 | COM235 | Thanks for joining the Student Zone at Build | [Chloe Condon](https://twitter.com/ChloeCondon), [Crazy Aunt Lindsey](https://www.thefablab.com/), [Liam McGregor](www.linkedin.com/in/liamjmcgregor/), [Morgan Dukes](https://www.linkedin.com/in/morgandukes3/), [Swetha Prabakaran](twitter.com/swethapraba) | [Microsoft the Musical](http://aka.ms/musical) |  |  |
 
 ## Try out Minecraft: Education Edition with MakeCode
-You can try out the Minecraft: Education Edition with these usernames:  
+
+You will need an Office 365 Education account to log in to the full version of Minecraft: Education Edition. If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins.
```
Comment (drguthals) [inline README.md]:
```suggestion
You will need an Office 365 Education account to log in to the full version of [Minecraft: Education Edition](https://education.minecraft.net/). If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins that we've provide below.
```

File: README.md [hunk 13]
```diff
@@ -56,17 +56,38 @@ Here are some examples of the content and star speakers we have this year:
 | COM235 | Thanks for joining the Student Zone at Build | [Chloe Condon](https://twitter.com/ChloeCondon), [Crazy Aunt Lindsey](https://www.thefablab.com/), [Liam McGregor](www.linkedin.com/in/liamjmcgregor/), [Morgan Dukes](https://www.linkedin.com/in/morgandukes3/), [Swetha Prabakaran](twitter.com/swethapraba) | [Microsoft the Musical](http://aka.ms/musical) |  |  |
 
 ## Try out Minecraft: Education Edition with MakeCode
-You can try out the Minecraft: Education Edition with these usernames:  
+
+You will need an Office 365 Education account to log in to the full version of [Minecraft: Education Edition](https://education.minecraft.net/). If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins that we've provide below.
+
+Enter your [school email address](https://education.minecraft.net/get-started/) to see if your school account is eligible for a trial. If it’s not, we’ll redirect you to a sign-up page.
```
Comment (drguthals) [inline README.md]:
```suggestion
To get started, enter your [school email address on the Minecraft: Education Edition website](https://education.minecraft.net/get-started/) to confirm that your school account is eligible for full access or even a free trial. Then, download the game and follow the sign in prompts from within the game, using your same school credentials.  

Additional support can be found on the [Help page for Minecraft: Education Edition](https://minecrafteducation.zendesk.com/). 
```

File: README.md [hunk 14]
```diff
@@ -56,17 +56,40 @@ Here are some examples of the content and star speakers we have this year:
...
-You can try out the Minecraft: Education Edition with these usernames:  
+
+You will need an Office 365 Education account to log in to the full version of [Minecraft: Education Edition](https://education.minecraft.net/). If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins that we've provide below.
+
+To get started, enter your [school email address on the Minecraft: Education Edition website](https://education.minecraft.net/get-started/) to confirm that your school account is eligible for full access or even a free trial. Then, download the game and follow the sign in prompts from within the game, using your same school credentials.  
+
+Additional support can be found on the [Help page for Minecraft: Education Edition](https://minecrafteducation.zendesk.com/). 
+
+
+### You can try out the Minecraft: Education Edition with these usernames:
```
Comment (drguthals) [inline README.md]:
```suggestion
### Demo accounts for Minecraft: Education Edition
If you do not have access to Minecraft: Education Edition through your school, you can try it out with these limited supply and limited time usernames:  
```

**Pull request merged**